gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+
+#### Standard Values
+
+We can imagine the following starting point of restrictions for use by systems that find them useful. In each case below, we provide a type definition for the object passed as the `value` for the restriction, as well as provide a suggested value for `key`.
+
+##### Language
+
+Keyed as `osdi:rider_restriction_language`:
+
+| Name | Type | Description
+|------|--------|------
+| iana_codes | string[] | Language codes, as defined by [RFC 5646](https://tools.ietf.org/html/rfc5646), that the associated rider can speak. Examples would include `en/US` or `es/419`.
+
+##### Accessibility
+
+Keyed as `osdi:rider_restriction_accessibility`
+
+| Name | Type | Description
+|------|--------|------
+| is_hearing_impaired | bool | Indicates that the Rider has a partial or total hearing impairment
+| is_vision_impaired | bool | Indicates that the Rider has a partial or total vision impairment
+| wheelchair_accessibility | enum* | If specified, one of `ramp`, indicating that the Rider requires a wheelchair ramp in order to enter or exit the Vehicle, or `collapsible` to indicate that no ramp is required, but that the associated Vehicle needs enough storage space to carry a collapsed wheelchair. Other variants may be added to this enum.
+
+### Vehicle Fields
+
+| Name | Type | Description
+|------|--------|------
+| id | string | The unique identifier for this vehicle
+| drivers | string[] | The Person(s) who are responsible for communicating with Riders of this vehicle and for its operation. In most cases this will be a singleton array, but it is possible that groups of people will together fulfill the driver role (example: One drives while the other navigates)
+| seating | VehicleSeat[] | An array of Seating values indicating how the passenger seating is configured
+| availability | Interval[] | An array of times during which this vehicle is operational and can be assigned to Demands.
+| range | Duration | The maximum amount of time this vehicle can travel for any particular ride. This value might be the minimum of the furthest the vehicle's driver is willing to travel, or of the maximum range of the vehicle itself, such as with an electric vehicle subject to frequent recharging stops.
+| location | Location | This vehicle's location, from which it is assumed to begin or end with each availiability interval.
+
```

Comment:
```suggestion
| license_plate | string [] | The vehicle's license plate number 
| vehicle_description | string [] | should include “color” “make” and “model” 
```
This would help when notifying riders: “Your driver is on the way in a blue Honda Civic. License plate: ABC123"